src/lexers/javascript-lexer.js

```diff
@@ -14,8 +18,21 @@ export default class JavascriptLexer extends BaseLexer {
   extract(content) {
     const that = this
 
+    var localAcorn = acorn;
+
+    if(this.acornOptions.plugins) {
+      if(this.acornOptions.plugins.stage3)
```

Comment:
To follow the project styleguide, please open/close the if statements with `{}`. Also this project doesn't use `;` at the end of statements.